Dfe.Data.SearchPrototype/Web/Tests/ViewTests/PaginationPresentationTests.cs

```diff
@@ -0,0 +1,378 @@
+﻿using AngleSharp;
+using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using Dfe.Data.SearchPrototype.SearchForEstablishments.ByKeyword.Usecase;
+using Dfe.Data.SearchPrototype.Web.Tests.Shared.Helpers;
+using Dfe.Data.SearchPrototype.Web.Tests.Shared.Pages;
+using Dfe.Data.SearchPrototype.Web.Tests.Shared.TestDoubles;
+using Dfe.Data.SearchPrototype.Web.Tests.ViewTests;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Moq;
+using Xunit;
+
+namespace Dfe.Data.SearchPrototype.Web.Tests.PresentationLayerTests;
+
+public class PaginationPresentationTests : SharedTestFixture
+{
+    private const string homeUri = "http://localhost";
+
+    public PaginationPresentationTests(WebApplicationFactory<Program> factory) : base(factory)
+    {
+    }
+
+    [Fact]
+    public async Task Pagination_WithCurrentPageInLowerPaddingBoundry_ShowsCorrectPaginationElements()
```

Comment:
typo **boundry**

Reply:
corrected- even if there was only one thing that i have learnt working on this it would be how to spell boundary lol